Assets/ModelReplacementSDK/Plugins/OffsetBuilder.cs

```diff
@@ -198,16 +199,26 @@ private IEnumerator SaveAssetBundle()
             initialized = true;
         }
 
-        private void MapSkeletonBones(SkeletonBone sk)
+        private void ForceRestPose(Animator animator)
         {
-            var matchingTransforms = GetComponentsInChildren<Transform>().Where(x => x.name == sk.name);
-            if (matchingTransforms.Any())
+            IEnumerable<Transform> childTransforms = animator.transform.root.GetComponentsInChildren<Transform>()
+                .Skip(1); // Skip root transform
+            foreach (var sk in animator.avatar.humanDescription.skeleton.Skip(1)) // Skip root bone
             {
-                matchingTransforms.First().localRotation = sk.rotation;
-            }
-            else
-            {
-                Debug.Log($"Missing bone {sk.name}");
+                var matchingTransforms = ListPool<Transform>.Get();
+                matchingTransforms.AddRange(childTransforms.Where(x => x.name == sk.name));
+                if (matchingTransforms.Count > 0)
+                {
+                    if (matchingTransforms.Count > 1)
+                    {
+                        Debug.LogWarning($"Ambiguous match: bone '{sk.name}' matches multiple transforms");
```

Comment:
Since all transforms are stored in the skeleton, it's possible for any arbitrary bone to match any arbitrary transform. This is likely to lead to undesired behavior, but there is no easy way to resolve which `SkeletonBone` matches a certain transform other than by name. At the very least we can print a warning when this happens to help the user figure out how to fix it.